crates/ruff/tests/snapshots/integration_test__rule_f401.snap

```diff
@@ -65,7 +84,8 @@ else:
 ```
 
 ## Options
-- `lint.ignore-init-module-imports`
+- Deprecated `lint.ignore-init-module-imports` to `true`. When set to `false`, unused imports
+in `__init__.py` files are removed (unsafe).
```

Comment:
I think these options might get dynamically turned into links during documentation generation, not sure if this will work. I'd just leave it as-is probably.

Reply:
Willfix